### ISSUE TITLE
Customising/Disabling Specific Models 

### ISSUE THREAD

**Issue by irenexu** [cosanlab/py-feat#216]
Hi!

Thank you for developing this fantastic toolkit! I am using it to detect facial action units and emotion changes in videos featuring only a single person's face, given that there is only one person in the videos, is it possible to disable the use of the identity_model and would that potentially decrease the processing time?

I am currently processing videos that average around 10 mins in length on an NVIDIA 4090 GPU and extracting per-frame data, any advice on optimising this process would be greatly appreciated.

**Comment by ljchang** [comment]
Hi @irenexu , we are close to completing a major refactor that will allow disabling individual models and also better performance on GPUs. stay tuned.

To speed things up, people have reported better performance by dividing videos into images using ffmpeg and parallelizing these jobs.

**Comment by ljchang** [comment]
Yes — in v0.7 you can disable models you don't need. For your single-person videos, pass `identity_model=None` to skip ArcFace identity entirely:

```python
from feat import Detector
detector = Detector(identity_model=None)
```

Other sub-models can likewise be disabled (e.g. `landmark_model=None`, `emotion_model=None`, `au_model=None`). Skipping identity avoids the per-face embedding pass and will reduce per-frame time. For long videos on a 4090 you may also want the new `Detectorv2` (single multitask forward for AU/emotion/pose/gaze/mesh). Closing as addressed in v0.7 — happy to reopen if disabling doesn't behave as expected.
